src/python/pants/backend/native/tasks/native_compile.py

```diff
@@ -185,10 +187,20 @@ def get_compiler(self):
   def _compiler(self):
     return self.get_compiler()
 
+  def get_third_party_include_dirs(self):
+    inc_dir = []
+    tp_files_product = self.context.products.get_data(NativeExternalLibraryFetch.NativeExternalLibraryFiles)
+    directory = tp_files_product.include
+    if directory:
+      inc_dir = [directory]
+    return inc_dir
+
   def _make_compile_request(self, versioned_target, dependencies):
     target = versioned_target.target
     include_dirs = [self._include_dirs_for_target(dep_tgt) for dep_tgt in dependencies]
```

Comment:
Generally, if I'm mutating a variable, I isolate that along with the initialization of the variable and surround it by at least one line of whitespace on both sides, a habit I learned the hard way, and I would recommend doing that here and anywhere else this occurs.

Reply:
✅